utils/service_discovery/abstract_config_store.py

```diff
@@ -276,13 +297,21 @@ def get_check_tpls(self, identifier, **kwargs):
             # annotations for configs before falling back to autoconf.
             kube_annotations = kwargs.get(KUBE_ANNOTATIONS)
             kube_container_name = kwargs.get(KUBE_CONTAINER_NAME)
+            docker_labels = kwargs.get(DOCKER_LABELS)
+            source = ""
+
+            kube_config = None
```

Comment:
same here, you can call it `config` or whatever but it's not k8s specific anymore